src/test/java/hw2/ex2/SecondScenarioTest.java

```diff
@@ -0,0 +1,278 @@
+        WebElement actualUserName = driver.findElement(By.id("user-name"));
+        assertEquals(actualUserName.getText(), "PITER CHAILOVSKII");
+
+        // 5. Click on "Service" subcategory in
+        // the header and check that drop down contains options
+        // There is a clicking on "Service" header and getting actual
+        // header texts from service-dropdown menu. Then we create
+        // a list of expected text and compare them
+        driver.findElement(By.xpath("//li[@class='dropdown']")).click();
+        WebElement headerDropDownList = driver.findElement(By.
+                xpath("//ul[@class='dropdown-menu']"));
+        List<String> actualServiceHeaderTexts = Arrays.
+                asList(headerDropDownList.getText().split("\n"));
+        ArrayList<String> expectedServiceHeaderTexts = new ArrayList<>();
+        expectedServiceHeaderTexts.add("SUPPORT");
+        expectedServiceHeaderTexts.add("DATES");
+        expectedServiceHeaderTexts.add("SEARCH");
+        expectedServiceHeaderTexts.add("COMPLEX TABLE");
+        expectedServiceHeaderTexts.add("SIMPLE TABLE");
+        expectedServiceHeaderTexts.add("USER TABLE");
+        expectedServiceHeaderTexts.add("TABLE WITH PAGES");
+        expectedServiceHeaderTexts.add("DIFFERENT ELEMENTS");
+        expectedServiceHeaderTexts.add("PERFORMANCE");
+        assertEquals(actualServiceHeaderTexts, expectedServiceHeaderTexts);
+
+        // 6. Click on Service subcategory in
+        // the left section and check that drop down contains options
+        // There is a clicking on "Service" in the left section
+        // and getting actual texts from service-dropdown menu. Then
+        // we create a list of expected text and compare them
+        // NB! Soft Assert can be placed here 'cause we can
+        // continue test even if there elements aren't exist
+        // (we'll not use them in the further steps)
+        driver.findElement(By.className("menu-title")).click();
+        WebElement leftSectionDropDownList = driver.findElement(By.
+                xpath("//ul[@class='sub']"));
+        List<String> actualServiceLeftSectionTexts = Arrays.
+                asList(leftSectionDropDownList.getText().split("\n"));
+        ArrayList<String> expectedServiceLeftSectionTexts = new ArrayList<>();
+        expectedServiceLeftSectionTexts.add("Support");
+        expectedServiceLeftSectionTexts.add("Dates");
+        expectedServiceLeftSectionTexts.add("Complex Table");
+        expectedServiceLeftSectionTexts.add("Simple Table");
+        expectedServiceLeftSectionTexts.add("Search");
+        expectedServiceLeftSectionTexts.add("User Table");
+        expectedServiceLeftSectionTexts.add("Table with pages");
+        expectedServiceLeftSectionTexts.add("Different elements");
+        expectedServiceLeftSectionTexts.add("Performance");
+        sa.assertEquals(actualServiceLeftSectionTexts,
+                expectedServiceLeftSectionTexts);
```

Comment:
1. Creation of the `List` could be simplified
2. Using split("\n") not good idea(

Reply:
Solved (as in previous item):
1. I use Arrays.asList now, it looks more simple
2. I get all <*li*> from that list now and added them to my ArrayList